templates/docker/php/bin/magento-configure

```diff
@@ -65,4 +55,21 @@ $contents = <<<FILE
 return %s;
 FILE;
 
-file_put_contents('app/etc/env.php', sprintf($contents, var_export($config, true)));
\ No newline at end of file
+file_put_contents('app/etc/env.php', sprintf($contents, var_export($config, true)));
+
+# Configure Varnish
+$sql = <<<SQL
+DELETE FROM `core_config_data` WHERE `path` like 'system/full_page_cache%';
+INSERT INTO `core_config_data` (`scope`, `scope_id`, `path`, `value`)
+VALUES
+	('default', 0, 'system/full_page_cache/caching_application', '2'),
+	('default', 0, 'system/full_page_cache/varnish/access_list', 'php, nginx, varnish'),
+	('default', 0, 'system/full_page_cache/varnish/backend_host', 'nginx'),
+	('default', 0, 'system/full_page_cache/varnish/backend_port', '80'),
+	('default', 0, 'system/full_page_cache/varnish/grace_period', '300');
+SQL;
```

Comment:
As mentioned in Slack, I think we could be using `bin/magento config:set` or the n98 equivalent for this rather than raw SQL (although this is probably more performant and less problematic if e.g. cache is built incorrectly)